_data/domains.yml

```diff
@@ -438,6 +438,13 @@ websites:
       - software
     doc: https://www.pairdomains.com/kb/posts/345
 
+  - name: Park.io
+    url: https://park.io/
+    img: parkio.png
+    tfa:
+      - software
+    doc: https://park.io/support
```

Comment:
```suggestion
```

Reply:
This link points to a FAQ page with the following question: "Can I use multi-factor authentication to add security to my Park.io account?".

Unfortunately there's no way to link directly to this question (no anchor in HTML).

I suggest we keep the link - it's better than nothing...

Reply:
@Carlgo11 Can you confirm leaving the doc link as-is, lacking a better option?

Reply:
Yeah it’s fine.